site/content/en/docs/tasks/manage/setup_failure_recovery.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Setup failure recovery"
+date: 2025-12-04
+weight: 12
+description: >
+  Configure failure recovery mechanisms to unblock workloads after a node outage.
+---
+
+This describes and shows how to configure the failure recovery mechanism that automatically transitions pods
+that are stuck terminating into the `Failed` phase. This is especially relevant for `Job`-based workloads that
+specify [`podReplacementPolicy: Failed`](https://kubernetes.io/docs/concepts/workloads/controllers/job/#pod-replacement-policy),
+preventing the `Job` from making progress if such scenario occurrs.
+
+Without the failure recovery controller, a network partition or a malfunction of the `kubelet` (or a more general node failure) results in the pods running on that node to become stuck.
+The failure recovery mechanism unblocks workloads relying on pods being in a terminated state by forcefully moving them to the `Failed` phase after
+a **grace period of 60 seconds** since they became stuck.
+
+{{% alert title="Important" color="warning" %}}
+
+The default Kubernetes behaviour of keeping the pods `Terminating` is a deliberate safety measure, as in that scenario the control plane has no way to confirm that the pods actually stopped
+and released all their resources.
+Before opting into the feature, make sure the affected workloads will not cause any data corruption if re-scheduled without cleanup.
+
+{{% /alert %}}
+
+# Prerequisites
+
+- A running Kueue installation at **v0.15** or newer.
+- The `FailureRecoveryPolicy` feature-gate enabled in the Kueue controller manager. Check the [Installation](/docs/installation/#change-the-feature-gates-configuration) guide for details on feature gate configuration.
```

Comment:
"feature gate" seems much more widespread than "feature-gate".

Reply:
Nevermind! I did not notice the first occurrence of "feature-gate". I changed this, although this was copied over from https://github.com/kubernetes-sigs/kueue/blob/main/site/content/en/docs/tasks/manage/setup_object_retention_policy.md#prerequisites